src/webapp/gradebookServer.js

```diff
@@ -190,12 +190,24 @@ app.get('/seasons', function(request, response) {
       passwordText, request.query.host, request.query.port);
 
    //Get the params from the url
-   var instructorID = request.query.instructorid;
+   var userID = request.query.userid;
+   var userRole = request.query.userRole;
    var year = request.query.year;
-
-   //Set the query text
-   var queryText = 'SELECT SeasonOrder, SeasonName FROM gradebook.getInstructorSeasons($1, $2);';
-   var queryParams = [instructorID, year];
+   var queryText;
+   var queryParams;
+
+   //Set the query
+   var queryParams = [userID, year];
+
+   if(userRole == 'alpha_GB_Instructor') {
+      queryText = 'SELECT Order, Name FROM getInstructorSeasons($1, $2);';
+   }
+   else if(userRole == 'alpha_GB_Student') {
+      queryText = 'SELECT Order, Name FROM getStudentSeasons($1, $2);';
```

Comment:
```suggestion
      queryText = 'SELECT SeasonOrder, SeasonName FROM getSeasonsAsStudent();';
```

See previous comment for the change in parameter names. The function call was changed as students are not allowed to call `getStudentSeasons()`

Reply:
Actually, this also means that the `queryParams` variable needs to be set to different values depending on the query called. See the most recent version of #38 for an example.